### PR TITLE
News Functionality

### PR DESCRIPTION
**Pull request by Azudeus**
Client view for news hasn't well designed
There is no admin template yet, so to open admin view open
/News/List

### CODE REVIEW

File: app/views/App/News.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{set . "title" "ITB Career Center Entrepreneurship News"}}
-
+{{set . "list" .list}}
```
Comment (ParadiseCatz) [inline app/views/App/News.html]: This is not necessary

File: app/views/News/Detail.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{{set . "news" .news}}
```
Comment (ParadiseCatz) [inline app/views/News/Detail.html]: This is not necessary

File: app/views/App/News.html (hunk 1 above)
Comment (ParadiseCatz) [reply app/views/App/News.html]: Actually you don't need to redefine the parameter from Render()

File: app/views/header.html [hunk 3]
```diff
@@ -9,6 +9,7 @@
 		<link rel="stylesheet" href="/public/assets/css/main.css" />
 		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
 		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
+		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (ParadiseCatz) [inline app/views/header.html]: If possible please use the stylesheet locally, since the application would be deployed inside ITB network

**Comment by Azudeus** [comment]
Fixed unnecesary parameter and local css

**Comment by ParadiseCatz** [comment]
Looks good. Approved.

**Pull request merged**